Off-Top-App/off-top-flutter
# OT-98 | FE - Properly convert audio data into a audio file in Spring-Boot

**What I did:**
- Made our components more credible inside of RecordingSession.dart
- Abstracted the WebSocket component to be a class rather than a whole page
- Am now connecting to the Websocket when beginning our recording session

**How To Test:**
- Make sure that your kafka/zookeeper/spring-boot server's are running properly.
- When you enter the recordingSession page and click on the recorder icon you should see that you have created a session in the spring-boot server
- When you end your session you should see in in Spring-Boot that you have produced the message
